webapp/src/constants/links.tsx

```diff
@@ -322,4 +322,14 @@ export class LINKS {
   );
 
   static PROJECT_IMPORT = Link.ofParent(LINKS.PROJECT, 'import');
+
+  static PROJECT_DEVELOPER = Link.ofParent(LINKS.PROJECT, 'developer');
```

Comment:
I was thinking that we should probably add the `content delivery configs` path as well, although its a default path for developer, since it's semantic. For example if we want to link there from the documentation, feature changing of the default for developers path (possibly because of adding some cool developer dashboard or something) would break such link.

Reply:
👍